analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DisposeFromDispose.CSharp10.cs

```diff
@@ -19,10 +19,19 @@ record struct R1 : IDisposable
     public void Dispose() { }
 }
 
+record struct R2 : IDisposable
+{
+    private FileStream Fs { get; set; }
+
+    public void OpenResource(string path) => this.Fs = new FileStream(path, FileMode.Open);
+    public void CloseResource() => this.Fs.Close();
+    public void CleanUp() => this.Fs.Dispose(); // Compliant. Properties are not not considered.
```

Comment:
This probably is a FN. I added the test case, so it isn't missed. We may turn it into an FN and add an issue.